unitTests/magicbox/SuggestionsManagerTest.ts

```diff
@@ -329,11 +329,11 @@ export function SuggestionsManagerTest() {
       });
 
       it("sets aria-owns of InputManager to the id of SuggestionManager's listbox", () => {
-        expect(inputManager.input.getAttribute('aria-owns')).toEqual(suggestionListboxID);
+        expect(inputManager.input.getAttribute('aria-owns')).toMatch(/coveo-magicbox-suggestions/);
       });
 
       it("sets aria-controls of InputManager to the id of SuggestionManager's listbox", () => {
-        expect(inputManager.input.getAttribute('aria-controls')).toEqual(suggestionListboxID);
+        expect(inputManager.input.getAttribute('aria-controls')).toMatch(/coveo-magicbox-suggestions/);
```

Comment:
Shouldn't we check for the id in this test and the one above? If not, then the test descriptions no longer reflect the test.